src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,27 +28,16 @@
  */
 public class RobotContainer {
     // Subsystems
-    //private final SwerveSubsystem swerveSubsystem;
-    private final MissileShellSwerveSubsystem swerveSubsystem;
+    // private final SwerveSubsystem swerveSubsystem;
+    final GripperSubsystem gripper = new GripperSubsystem();
```

Comment:
Package private 😳

Reply:
Deleting the boilerplate 😔 

<img width="635" alt="image" src="https://user-images.githubusercontent.com/60120929/207529811-ef466527-c315-492f-8362-94d83ce0926f.png">